.github/workflows/ci-deep.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       MAINNET_RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
     uses: "sablier-labs/reusable-workflows/.github/workflows/forge-test.yml@main"
     with:
-      foundry-fuzz-runs: ${{ fromJSON(inputs.forkFuzzRuns) || 1000 }}
+      foundry-fuzz-runs: ${{ fromJSON(inputs.forkFuzzRuns || '1000') }}
       foundry-profile: "test-optimized"
-      match-path: "test/fork/**/*.sol"
+      match-path: "test/{core,periphery}fork/**/*.sol"
```

Comment:
I think we should separate out core and periphery runs? For example 

```yaml
core-test-fork:
  needs: ["lint", "build"]
  secrets:
    MAINNET_RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
  uses: "sablier-labs/reusable-workflows/.github/workflows/forge-test.yml@main"
  with:
    foundry-fuzz-runs: 20
    foundry-profile: "test-optimized"
    match-path: "test/core/fork/**/*.sol"
    name: "Core fork tests"

periphery-test-fork:
  needs: ["lint", "build"]
  secrets:
    MAINNET_RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
  uses: "sablier-labs/reusable-workflows/.github/workflows/forge-test.yml@main"
  with:
    foundry-fuzz-runs: 20
    foundry-profile: "test-optimized"
    match-path: "test/periphery/fork/**/*.sol"
    name: "Periphery fork tests"
```

So when a test fails, it would be easy to debug.

Reply:
Doing so it would be verbose, how can it improve debugging? You still can see the test fail/test path:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/0323ea0b-0258-4347-ad94-a9bc49c03812">

<img width="400" alt="image" src="https://github.com/user-attachments/assets/eaaa236c-2105-4e76-8596-a9183d6ff8a4">

Reply:
Agree that it wont improve debugging. I actually wanted to make the flow looks like the following:

<img width="464" alt="Screenshot 2024-07-30 at 10 40 53" src="https://github.com/user-attachments/assets/9587121b-deb8-4c62-8810-93e6fc0fc544">

where we have separate boxes for core and periphery. I thought it would look cool. But I couldn't figure out how to do it through yaml file. Marking it resolve as I agree with you.